src/oc_erchef/apps/chef_index/c_src/cs_escape.c

```diff
@@ -0,0 +1,224 @@
+/* Copyright 2015 Chef Software, Inc.
```

Comment:
I combed through this pretty thoroughly earlier in the week - any changes since then?